data/json/items/comestibles/drink.json

```
@@ -1041,5 +1041,63 @@
     "proportional": { "quench": 1.2 },
     "relative": { "fun": 1 },
     "use_action": [  ]
+  },
+  {
+    "type": "COMESTIBLE",
+    "id": "coffee_sweetened",
+    "copy-from": "coffee",
+    "name": "sweetened coffee",
+    "calories": 21,
+    "description": "The morning ritual of the pre-apocalyptic world, created from coffee cherries through a complex process of seed removal, roasting, grinding, and brewing.  Coffee is substantially richer in caffeine than its rival tea. With added sweeter for better taste.",
+    "price": 110,
+    "price_postapoc": 60,
+    "material": "water",
+    "fun": 10
+  },
+  {
+    "type": "COMESTIBLE",
+    "id": "tea_sweetened",
+    "copy-from": "tea",
+    "name": "sweetened tea",
+    "calories": 21,
+    "description": "The beverage of gentlemen everywhere, made from applying hot water to leaves of the tea plant /Camellia sinensis/. Added sweeter for a better taste.",
+    "price": 100,
+    "price_postapoc": 35,
+    "fun": 10
+  },
+  {
+    "type": "COMESTIBLE",
+    "id": "milk_tea_sweetened",
+    "copy-from": "milk_tea",
+    "name": "sweetened milk tea",
+    "calories": 88,
+    "description": "Hot tea with cold milk and added sweeter.",
+    "price": 460,
+    "price_postapoc": 60,
+    "vitamins": [ [ "vitA", 5 ], [ "vitB", 3 ], [ "vitC", 2 ], [ "calcium", 12 ] ],
+    "fun": 14
+  },
+  {
+    "type": "COMESTIBLE",
+    "id": "coffee_substitute_sweetened",
+    "copy-from": "coffee_substitute",
+    "name": "sweetened coffee substitute",
+    "calories": 21,
+    "description": "Homemade not-coffee created from the Kentucky coffeetree, just like the Meskwaki tribe!  Doesn't actually have any caffeine, and is very bitter, but it'll pass in a pinch. The added sweetness neutralize the bitterness somewhat.",
+    "price": 110,
+    "price_postapoc": 35,
+    "flags": [ "EATEN_HOT", "NUTRIENT_OVERRIDE" ],
+    "fun": 1
+  },
+  {
+    "type": "COMESTIBLE",
+    "id": "milk_coffee_sweetened",
+    "copy-from": "milk_coffee",
+    "name": "sweetened coffee milk",
+    "calories": 88,
+    "description": "Coffee syrup mixed into milk.  It's been the state drink of Rhode Island since 1993. Added sweeter for those who like it even sweeter.",
```

Here too. Github glitched for this one somehow.
```suggestion
    "description": "Coffee syrup mixed into milk.  It's been the state drink of Rhode Island since 1993.  Added sweeter for those who like it even sweeter.",
```

---

Ok, but I explicitly linted it so it should have catched that, no?

---

No, the linter does not change the data, it only formats the JSON lines.